packages/sui-ssr/server/index.js

```diff
@@ -85,7 +85,7 @@ let _memoizedHtmlTemplatesMapping = {}
         customTlds: /localhost|\.local/
       })
 
-      !parsedUrl || parsedUrl.tld === 'localhost' // eslint-disable-line
+        !parsedUrl || parsedUrl.tld === 'localhost' // eslint-disable-line
```

Comment:
Is the linter working correctly? Some changes seems to be off.

Reply:
I dont know, something with the linter. It is prettier.